packages/@ama-sdk/create/src/index.ts

```diff
@@ -73,7 +73,9 @@ const schematicArgs = [
 const resolveTargetDirectory = resolve(process.cwd(), targetDirectory);
 
 const run = () => {
-  const isSpecPathUrl = url.URL.canParse(argv['spec-path']);
+  const isSpecPathUrl = !!argv['spec-path']
+    && url.URL.canParse(argv['spec-path'])
+    && (new url.URL(argv['spec-path'])).protocol.startsWith('http');
```

Comment:
If absolute path `url.URL.canParse` returns true
so it will not work anymore with this check

Reply:
absolute paths make `url.URL.canParse` returning `true` on with but in that case `protocol` will return the hardrive letter identifier.
It does not really break this usecase

Reply:
Later we call ` relative(resolveTargetDirectory, resolve(process.cwd(), argv['spec-path']))` (L93)
if `specPath` is an absolute path it will work correctly ?

Reply:
Ooooh I see, I haven't check the file indeed and has been abused by the variable name.
I will fix the misleading name and rollback the change